src/draw/vg_lite/lv_vg_lite_utils.c

```diff
@@ -1063,12 +1068,17 @@ void lv_vg_lite_finish(struct _lv_draw_vg_lite_unit_t * u)
 
     LV_VG_LITE_CHECK_ERROR(vg_lite_finish());
 
+    LV_LOG_USER("/// flush_count: %d", u->flush_count);
```

Comment:
I guess this line is not needed, only a debugging left-over.
Or if still needed, then correct text (remove `///` )